plugins/httpapi/nd.py

```diff
@@ -232,6 +233,8 @@ def send_file_request(self, method, path, file=None, data=dict(), remote_path=No
                             remote_path="/temp",
                         )
         """
+        if data is None:
+            data = dict()
```

Comment:
Does, this additional change make any difference?

Reply:
This was a sanity test failing because you not allowed to set dict() as default paramter: dangerous dict violation rule.